apps/src/templates/levelSummary/summary.module.scss

```diff
@@ -98,23 +105,39 @@ label.sectionSelector {
     column-gap: 48px;
     padding-top: 24px;
     padding-bottom: 12px;
+
+    .studentResponseBlock {
+      margin-bottom: 16px;
+    }
+
+    .studentName {
+      min-height: 24px;
+
+      p {
+        margin: 0;
+        font-size: 14px;
+        font-weight: bold;
+        padding-left: 12px;
+      }
+    }
   }
 
   .studentAnswer {
-    padding: 12px;
```

Comment:
Moved these to the "interior" to ensure the wrapping kept the names included in the wrapping.